task_23/dima.gres/src/main/java/NoExpress.java

```diff
@@ -0,0 +1,6 @@
+public class NoExpress extends Throwable {
```

Comment:
Don't extend Throwable.
Choose between Exception - for checked and RuntimeException - for unchecked.